packages/office-ui-fabric-react/src/components/Button/DefaultButton.visualtest.ts

```diff
@@ -0,0 +1,29 @@
+import { Casper, IPhantomCSS } from '../../visualtest/PhantomCssInterface';
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.
+  start(baseUrl + 'defaultButton').
+  then(function () {
```

Comment:
No they should not be lambda functions because casper explicitly expects `this` to be passed through functions.